src/network/crypto.ts

```diff
@@ -25,7 +25,7 @@ export async function decryptFileFromFs(
   },
 ): Promise<void> {
   const decipher = getAes256CtrDecipher(fileDecryptionKey.slice(0, 32), iv);
-  const fileSize = parseInt((await stat(fromPath)).size);
+  const fileSize = (await stat(fromPath)).size;
```

Comment:
last time I was working with this codebase, the size field was a number stringified, is not the case anymore?

Reply:
Not anymore since react-native-fs is updated too

https://github.com/itinance/react-native-fs/blob/master/index.d.ts#L23